docs/development/style_guide.rst

```diff
@@ -0,0 +1,123 @@
+
+Therefore, `Snake Case <https://en.wikipedia.org/wiki/Snake_case>`_ is preferred.
+For this reason, RocketPy is being converted from Camel to Snake as of version ``1.0.0``.
+New contributions should strive to follow Snake case as well.
+
+Furthermore, when it comes to naming new variables, functions, classes or anything in RocketPy, always try to use descriptive names.
+
+As an example, instead of using `a` or `alpha` as a variable for a rocket's angle of attack, `angle_of_attack` is preferred.
+Such descriptive names make the code significantly easier to understand, review and maintain.
+
+In summary:
+
+.. code-block:: python
+
+    # Not acceptable
+    a = 0.2 # angle of attack
+
+    # Preferred
+    angle_of_attack = 0.2 # in rad
+
+
+Linting
+^^^^^^^
+As far as line wrapping, parentheses, calling chains and other code style related matter goes, RocketPy currently employs `Black style <https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html>`_.
+Installing Black and running it before submitting a pull request is highly recommend.
+Currently, there are pull request tests in place to enforce that Black style is used.
+
+
+Documentation Style
+-------------------
+
+Every class, method, attribute and function added to RocketPy should be well documented using docstrings.
+RocketPy follows NumPy style docstrings, which are very well explained here: `NumPyDoc Style Guide <https://numpydoc.readthedocs.io/en/latest/format.html>`_.
+
+Here is a short example of the simplest acceptable docstring:
+
+
+Git Style
+---------
+
+Branch Naming Conventions
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+RocketPy tries to follow the `GitHub Flow <https://guides.github.com/introduction/flow/>`_ convention, keeping it simple!
+However, we aren't exactly strict about it.
+So here are a couple of **guidelines** to help you when creating new branches to contribute to the project:
+
+* Use branch names that follow the ``type/description`` convention.
+* ``type`` can be one of the following:
+
+    #. ``bug``: when your branch attempts to fix a bug
+    #. ``doc``: when your branch adds documentation changes
+    #. ``enh``: when you add new features and enhancements
+    #. ``maint``: when your branch is all about refactoring, fixing typos, etc.
+    #. ``rel``: when your branch makes changes related to creating new releases
+
+* Use ``-`` instead of spaces for the description text.
+* Keep branch names with lowercase letters.
+* Reference issue numbers and context if relevant.
+
+Here are a couple of example branch names:
+
+- ``maint/refactor-parachute-implementation``
+- ``bug/issue-98-upside-down-rockets``
+- ``enh/hybrid-motor-feature``
+- ``maint/typos-flight-class``
+
+Once you are ready to create a Pull Request for your branch, we advise you to merge with the ``dev`` branch instead of the default ``master`` branch.
+This way, we keep the ``master`` branch stable and use the ``dev`` branch to test out new features!

```

Comment:
Just a heads up, under [Git Style→Branch Naming Conventions] we refer to our development branch as `dev`, while it is currently named `develop` (more frequently used). Might be worth altering it to avoid confusion.

Reply:
Thanks for the heads up! Fixed it!